Jenkinsfile.aws-test

```diff
@@ -0,0 +1,73 @@
+def NODE = "rhcos-jenkins"
+def AWS_REGION = "us-east-1"
+
+// this var conveniently refers to a location on the server as well as the
+// local dir we sync to/from
+def images = "/srv/rhcos/output/images"
+
+node(NODE) {
+    stage("Clean workspace") {
+       step([$class: 'WsCleanup'])
+    }
+    checkout scm
+    utils = load("pipeline-utils.groovy")
+
+    // This job should only be triggered by Jenkinsfile.cloud
+    utils.define_properties(null)
+
+    if (params.DRY_RUN) {
+        echo "DRY_RUN set, skipping aws tests and launch permissions"
+        currentBuild.result = 'NOT_BUILT'
+        currentBuild.description = '(dry run)'
+        return
+    }
+
+    utils.inside_assembler_container("-v /srv:/srv") {
+    stage("Sync In") {
+        withCredentials([
+            string(credentialsId: params.ARTIFACT_SERVER, variable: 'ARTIFACT_SERVER'),
+            sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
+        ]) {
+            sh "mkdir -p ${images}"
+            utils.rsync_dir_in(ARTIFACT_SERVER, KEY_FILE, images)
+        }
+    }
+
+    try {
+    stage("Run Kola tests on intermediate aws image") {
```

Comment:
It would be helpful to set the `currentBuild.description` to the AMI ID near the beginning of this stage.  Kind of like this:

https://github.com/openshift/os/blob/master/Jenkinsfile.cloud#L125

Reply:
Added below!